examples/engine/3_responses.py

```diff
@@ -59,57 +68,105 @@ def cannot_talk_about_topic_response(ctx: Context, actor: Actor, *args, **kwargs
+        "start_node": {  # This is an initial node,
+            # it doesn't need a `RESPONSE`.
             RESPONSE: "",
-            TRANSITIONS: {"node1": cnd.exact_match("Hi")},  # If "Hi" == request of user then we make the transition
+            TRANSITIONS: {"node1": cnd.exact_match("Hi")},
+            # If "Hi" == request of user then we make the transition
         },
         "node1": {
-            RESPONSE: rsp.choice(["Hi, what is up?", "Hello, how are you?"]),  # random choice from candicate list
-            TRANSITIONS: {"node2": cnd.exact_match("i'm fine, how are you?")},
+            RESPONSE: rsp.choice(["Hi, what is up?", "Hello, how are you?"]),
+            # Random choice from candicate list.
+            TRANSITIONS: {"node2": cnd.exact_match("I'm fine, how are you?")},
         },
         "node2": {
             RESPONSE: "Good. What do you want to talk about?",
             TRANSITIONS: {"node3": cnd.exact_match("Let's talk about music.")},
         },
-        "node3": {RESPONSE: cannot_talk_about_topic_response, TRANSITIONS: {"node4": cnd.exact_match("Ok, goodbye.")}},
-        "node4": {RESPONSE: upper_case_response("bye"), TRANSITIONS: {"node1": cnd.exact_match("Hi")}},
-        "fallback_node": {  # We get to this node if an error occurred while the agent was running
+        "node3": {
+            RESPONSE: cannot_talk_about_topic_response,
+            TRANSITIONS: {"node4": cnd.exact_match("Ok, goodbye.")},
+        },
+        "node4": {
+            RESPONSE: upper_case_response("bye"),
+            TRANSITIONS: {"node1": cnd.exact_match("Hi")},
+        },
+        "fallback_node": {  # We get to this node
+            # if an error occurred while the agent was running.
             RESPONSE: fallback_trace_response,
             TRANSITIONS: {"node1": cnd.exact_match("Hi")},
         },
     }
 }
 
-
 # testing
 happy_path = (
     ("Hi", "Hello, how are you?"),  # start_node -> node1
-    ("i'm fine, how are you?", "Good. What do you want to talk about?"),  # node1 -> node2
-    ("Let's talk about music.", "Sorry, I can not talk about music now."),  # node2 -> node3
+    (
+        "I'm fine, how are you?",
+        "Good. What do you want to talk about?",
+    ),  # node1 -> node2
+    (
+        "Let's talk about music.",
+        "Sorry, I can not talk about music now.",
+    ),  # node2 -> node3
     ("Ok, goodbye.", "BYE"),  # node3 -> node4
     ("Hi", "Hi, what is up?"),  # node4 -> node1
-    ("stop", {"previous_node": ("greeting_flow", "node1"), "last_request": "stop"}),  # node1 -> fallback_node
-    ("one", {"previous_node": ("greeting_flow", "fallback_node"), "last_request": "one"}),  # f_n->f_n
-    ("help", {"previous_node": ("greeting_flow", "fallback_node"), "last_request": "help"}),  # f_n->f_n
-    ("nope", {"previous_node": ("greeting_flow", "fallback_node"), "last_request": "nope"}),  # f_n->f_n
+    (
```

Comment:
yep, now test is so ugly. I think we can fix it in future. https://trello.com/c/DmtDGMM0